.gitignore

```
@@ -0,0 +1,6 @@
+.vscode/settings.json
+.DS_Store
+__pycache__
```

Yo cambiaría la linea 3 por eso, así nos aseguramos de no incluir ningún archivo de bytecode autogenerado, no son necesarios..
```suggestion
__pycache__/
*.pyc
```